docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+##
+
+.. raw:: html
+
+   <center>
+
+:math:`\mathbf{\text{Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations}}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Ge, Z.M. and Cheng, Y.H., 1982. Extended Kane’s equations for nonholonomic variable mass system.}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Figure 1: Idealized model of a jet racing car that is propelled by a rocket engine at point P, the rocket engine being treated as a variable mass particle at P}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Here}, \{a_x: g_3, a_y: g_1, a_z: g_2\}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Leimanis, E., 2013. The general problem of the motion of coupled rigid bodies about a fixed point (Vol. 7). Springer Science & Business Media. (Page 207)}`
+
+.. raw:: html
+
+   </center>
+
+.. code:: ipython3
+
+    import sympy as sm
+    import sympy.physics.mechanics as me
+    from pydy.system import System
+    import numpy as np
+    from sympy.simplify.fu import TR2
+    import matplotlib.pyplot as plt
+    from scipy.integrate import odeint
+    me.init_vprinting()
+
+.. code:: ipython3
+
+    N = me.ReferenceFrame('N')
+    
+    q1, q2, q3, q4, q5, q6, q7, q8 = me.dynamicsymbols('q1:9')
+    
+    A2 = N.orientnew('A_2', 'Axis', [q3, N.z])
+    A1 = A2.orientnew('A_1', 'Axis', [q4, A2.z])
+    
+    B1 = A1.orientnew('B_1', 'Axis', [q5, A1.y])
+    B2 = A1.orientnew('B_2', 'Axis', [q6, A1.y])
+    B3 = A2.orientnew('B_3', 'Axis', [q7, A2.y])
+    B4 = A2.orientnew('B_4', 'Axis', [q8, A2.y])
+    
+    t = me.dynamicsymbols._t
+
+.. code:: ipython3
+
+    O = me.Point('O') # fixed point in the inertial reference frame
+    O.set_vel(N, 0)
+
+.. code:: ipython3
+
+    L, l , a, b, r1, r2 = sm.symbols('L, l , a, b, r_1, r_2')
+
+.. code:: ipython3
+
+    Q = O.locatenew('Q', q1 * N.x + q2 * N.y)
+
+.. code:: ipython3
+
+    P = Q.locatenew('P', L * -A2.x)
+
+.. code:: ipython3
+
+    C = P.locatenew('C', l * A2.x)
+
+.. code:: ipython3
+
+    Q.set_vel(N, Q.pos_from(O).dt(N))
+    Q.vel(N)
+
+
+
+
+.. math::
+
+    \displaystyle \dot{q}_{1}\mathbf{\hat{n}_x} + \dot{q}_{2}\mathbf{\hat{n}_y}
```

Comment:
No code output should be included. https://github.com/jupyter/jupyter-sphinx will create the outputs when sphinx is run to build the documentation website.